.vscode/settings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "editor.rulers": [
+    80
+  ],
+  "editor.tabSize": 2,
+  "javascript.validate.enable": true,
+  "search.exclude": {
+    "**/node_modules": true,
+    "**/graphql": true,
+    ".cache/**": true
+  },
+  "files.exclude": {
+    "**/.git": false
+  },
+  "[json]": {
+    "editor.formatOnSave": false
+  },
+  "[yaml]": {
+    "editor.formatOnSave": false
+  },
+  "prettier.singleQuote": false,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "typescript.preferences.importModuleSpecifier": "non-relative"
```

Comment:
Remove

Reply:
I've noticed that absolute import paths can break libraries that are imported into consumers -- this ok here? (Fine with it as long as its ok and nothing seems amiss in other apps)

Reply:
Ah, this wasn't removed? Yeah we should remove this.

Reply:
Removed it.